src/Microsoft.PowerShell.Commands.Utility/commands/utility/UnblockFile.cs

```diff
@@ -1,18 +1,19 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
@TravisEz13, your last commit had 3 failures in `PowerShell-CI-macos`
Verify approved aliases list.All approved Cmdlets present (no new Cmdlets added, no Cmdlets removed)
```powershell
Expected $null or empty, but got Unblock-File.
at <ScriptBlock>, /Users/runner/runners/2.160.1/work/1/s/test/powershell/engine/Basic/DefaultCommands.Tests.ps1: line 584
584:         $result | Should -BeNullOrEmpty
```
Verify approved aliases list.All present Cmdlets should have the correct ConfirmImpact
```powershell
Expected $null or empty, but got @(@{Name=Update-Help; ConfirmImpact=Medium; SideIndicator=<=}).
at <ScriptBlock>, /Users/runner/runners/2.160.1/work/1/s/test/powershell/engine/Basic/DefaultCommands.Tests.ps1: line 606
606:         $result | Should -BeNullOrEmpty
```
Unimplemented Utility Cmdlet Tests.Unblock-File should only be available on Windows
```powershell
Expected $false, but got $true.
at <ScriptBlock>, /Users/runner/runners/2.160.1/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/Unimplemented-Cmdlet.Tests.ps1: line 12
12:             [bool](Get-Command $Command -ErrorAction SilentlyContinue) | Should -Be $IsWindows
```